.github/workflows/ci.yml

```diff
@@ -126,6 +126,15 @@ jobs:
     uses: waku-org/nwaku/.github/workflows/container-image.yml@master
     secrets: inherit
 
+  QA_tests:
+    needs: build-docker-image
+    uses: waku-org/waku-interop-tests/.github/workflows/nim_waku_PR.yml@master
```

Comment:
Instead of using `master` I think it is better to use a fixed tag so that the changes in your `master` doesn't impact our CI :)

Reply:
done